README.md

```diff
@@ -19,27 +19,13 @@ It includes:
 
 To use this template docker and docker-compose need to be installed on your machine.
 
-1. Clone this repository.
-*Change `project-name` to your projects name.*
-```
-git clone <url> <project-name>
```

Comment:
So you don't think the git instructions are useful?

Reply:
Actually they might be for some people, I just assumed that this is common knowledge for whoever is browsing Github but I may have assumed wrong. I'll put it back.